packages/vue/src/examples/pages/login/Login.vue

```diff
@@ -1,114 +1,128 @@
+              @click="isLogIn = false"
+            >
+              Register now
+            </SfButton>
+          </div>
         </div>
-      </div>
-      <div
-        v-else
-        key="sign-up"
-        class="modal-content"
-        data-testid="signin-modal"
-      >
-        <form class="form" @submit.prevent="() => false">
-          <SfInput
-            v-model="firstName"
-            name="first-name"
-            label="Name"
-            class="form__element"
-          />
-          <SfInput
-            v-model="lastName"
-            name="last-name"
-            label="Last Name"
-            class="form__element"
-          />
-          <SfInput
-            v-model="email"
-            name="email"
-            label="Your email"
-            class="form__element"
-            type="email"
-          />
-          <SfInput
-            v-model="password"
-            name="password"
-            label="Password"
-            type="password"
-            class="form__element"
-          />
+        <div
+          v-else
+          key="sign-up"
+          class="modal-content"
+          data-testid="signin-modal"
+        >
+          <form class="form" @submit.prevent="() => false">
+            <SfInput
+              v-model="firstName"
+              name="first-name"
+              label="Name"
+              class="form__element"
+            />
+            <SfInput
+              v-model="lastName"
+              name="last-name"
+              label="Last Name"
+              class="form__element"
+            />
+            <SfInput
+              v-model="email"
+              name="email"
+              label="Your email"
+              class="form__element"
+              type="email"
+            />
+            <SfInput
+              v-model="password"
+              name="password"
+              label="Password"
+              type="password"
+              class="form__element"
+            />
+            <SfButton
+              type="submit"
+              class="sf-button--full-width form__submit"
+              data-testid="create-acount-button"
+            >
+              Create an account
+            </SfButton>
+          </form>
           <SfButton
-            type="submit"
-            class="sf-button--full-width form__submit"
-            data-testid="create-acount-button"
+            class="sf-button--text action-button"
+            data-testid="log-in-account"
+            @click="isLogIn = true"
           >
-            Create an account
+            or Log In To Your Account
           </SfButton>
-        </form>
-        <SfButton
-          class="sf-button--text action-button"
-          data-testid="log-in-account"
-          @click="isLogIn = true"
-        >
-          or Log In To Your Account
-        </SfButton>
-      </div>
-    </transition>
-  </SfModal>
+        </div>
+      </transition>
+    </SfModal>
+    <SfButton
+      style="margin: 0 auto"
```

Comment:
Let's create a class out of it. This is supposed to be code example that someone can copy and paste to their project so it should not contain inline styles.

Reply:
Okay I did.